app/src/main/java/com/locked/shingranicommunity/event/EventListViewModel.kt

```diff
@@ -191,6 +191,11 @@ class EventListViewModel @Inject constructor(
         fun onCleared() {
             eventItem.deObserveStatus(this::onStatusChanged)
         }
+        init {
+            eventItem.address?.let {
+                data.showMap.value = true
+            }
+        }
```

Comment:
This is good but there's already an `init {}` method at the beginning of the class. Just merge this with the one above.